go-samson-secret-puller/secrets.go

```diff
@@ -0,0 +1,36 @@
+package samsonsecretpuller
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+// Secrets returns a map of secret names and values
+func Secrets() (map[string]string, error) {
+	s, err := readSecrets("/secrets")
+	return s, err
```

Comment:
This can simply be `return readSecrets("/secrets")` since the return-values match.